webview-ui/src/index.css

```diff
@@ -1,44 +1,71 @@
-@import "tailwindcss";
+/* @import "tailwindcss"; */
+
+@layer theme, base, components, utilities;
+
+@import "tailwindcss/theme.css" layer(theme);
+/* https://tailwindcss.com/docs/preflight */
```

Comment:
CC @samhvw8 - When we added Tailwind some of the webview UI looked strange b/c Tailwind removes a lot of default styling via their preflight mechanism. I disabled it for now per https://tailwindcss.com/docs/preflight#disabling-preflight.